elevenlabs_api_benchmark.py

```diff
@@ -68,10 +77,12 @@
 general_group = parser.add_argument_group('General Parameters')
 general_group.add_argument("--voice_id", default=DEFAULT_VOICE_ID,
                            help="ID of the voice for speech synthesis")
+general_group.add_argument("--detect_audio_format", action='store_true',
+                           help="If set to True, the script checks the audio format and provides details like length, channels, and sample width of the audio")
 
 args = parser.parse_args()
 
-
+DETECT_AUDIO_FORMAT = args.detect_audio_format
```

Comment:
I'd recommend just using args.detect_audio_format in the code below. ALL_CAPS is usually for constants.